rclpy/test/test_rate.py

```diff
@@ -101,7 +101,8 @@ def test_rate_invalid_period(self):
 
     def test_destroy(self):
         rate = self.node.create_rate(FREQ)
-        self.node.destroy_rate(rate)
+        success = self.node.destroy_rate(rate)
```

Comment:
what about line 112? we should probably check on that as well?

Reply:
Indeed. Made it so.